File: 2578-split-with-minimum-sum/2578-split-with-minimum-sum.cpp
#include<string>
class Solution {
public:
    int splitNum(int num) {
        string s=to_string(num);
        sort(s.begin(), s.end());
        int a=0, b=0;
        for(int i=0; i<s.size(); i++) {
            if(i&1) {
                b*=10;
                b+=s[i]-'0';
            }
            else {
                a*=10;
                a+=s[i]-'0';
            }
        }
        return a+b;
    }
};

File: 2578-split-with-minimum-sum/2578-split-with-minimum-sum.js
/**
 * @param {number} num
 * @return {number}
 */
var splitNum = function(num) {
    
    let arr = num.toString().split('');
    arr.sort((a,b)=> a - b);
    let num1 = "";
    for(let i = 0; i<arr.length; i+=2){
        num1 += arr[i];
    }
    let num2 = "";
    for(let i = 1; i<arr.length; i+=2){
        num2 += arr[i];
    }
    return parseInt(num1) + parseInt(num2);
};